packages/vue/src/components/SpAssets/SpAssets.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <section>
+    <header class='assets-header'>
+      <h2 class='title'>Assets</h2>
+    </header>
+    <table class='assets-table'>
+      <tbody>
+      <tr v-for='balance in balances' :key='balance' class='assets-table__row'>
+        <td class='assets-table__denom'>
+          <div class="sp-denom-marker">
+            {{balance.denom.charAt(0).toUpperCase()}}
+          </div>
+          <div class='sp-denom-name'>
+            {{ balance.denom.toUpperCase() }}
+          </div>
+        </td>
+        <td class='assets-table__amount'>
+          {{ new Intl.NumberFormat('en-GB').format(balance.amount) }}
+        </td>
+      </tr>
+      </tbody>
+    </table>
+
+  </section>
+</template>
+
+<script lang="ts">
+/**
+ * SpAssets Component
+ *
+ * This component displays the current account balance.
+ */
+
+import { computed, defineComponent, onMounted, PropType } from 'vue'
+import { useStore } from "vuex"
+
+export default defineComponent({
+  name: 'SpAssets',
+  props: {
+    address: {
+      type: String as PropType<string>
+    }
+  },
+  setup(props) {
+    // Access store via useStore hook.
+    const store = useStore()
+
+    onMounted(async () => {
+      await store.dispatch('cosmos.bank.v1beta1/QueryAllBalances', {
```

Comment:
I would prefer having such dispatch wrapped around a local function like in [here](https://github.com/tendermint/vue/blob/release/milestone-01/packages/vue/src/components/SpAcc/SpAcc.vue#L197-L200)